advanced-analytics-framework-0.1.0-1.rockspec

```diff
@@ -23,10 +23,11 @@ dependencies = {
 build = {
    type = "builtin",
    modules = {
-      ["exasol.advanced_analytics_framework.exasol_script_tools"] = "exasol_advanced_analytics_framework/lua/src/exasol_script_tools.lua",
-      ["exasol.advanced_analytics_framework.query_loop"] = "exasol_advanced_analytics_framework/lua/src/query_loop.lua",
-      ["exasol.advanced_analytics_framework.query_handler_runner_main"] = "exasol_advanced_analytics_framework/lua/src/query_handler_runner_main.lua",
-      ["exasol.advanced_analytics_framework.query_handler_runner"] = "exasol_advanced_analytics_framework/lua/src/query_handler_runner.lua"
+      -- should the lua rock modules be renamed, too?
+      ["exasol.advanced_analytics_framework.exasol_script_tools"] = "exasol/analytics/lua/src/exasol_script_tools.lua",
+      ["exasol.advanced_analytics_framework.query_loop"] = "exasol/analytics/lua/src/query_loop.lua",
+      ["exasol.advanced_analytics_framework.query_handler_runner_main"] = "exasol/analytics/lua/src/query_handler_runner_main.lua",
+      ["exasol.advanced_analytics_framework.query_handler_runner"] = "exasol/analytics/lua/src/query_handler_runner.lua"
```

Comment:
Should the lua rock modules be renamed, too?

Reply:
If, then in another PR. And, probably later. I don't want to waste time with that at the moment